File: RUST/Cap04/c04ex05.rs
// c04ex05.rs

use std::io::{self, Write};

fn main() {
  let mes: i32;

  print!("Entre com um numero equivalente a um mes: ");
  io::stdout().flush().unwrap();
  let mut entra = String::new();
  io::stdin().read_line(&mut entra).expect("Falha ao ler a entrada");

  mes = entra.trim().parse().expect("Entrada inválida");

  match mes {
     1 => println!("Janeiro"),
     2 => println!("Fevereiro"),
     3 => println!("Marco"),
     4 => println!("Abril"),
     5 => println!("Maio"),
     6 => println!("Junho"),
     7 => println!("Julho"),
     8 => println!("Agosto"),
     9 => println!("Setembro"),
    10 => println!("Outubro"),
    11 => println!("Novembro"),
    12 => println!("Dezembro"),
     _ => println!("Mes invalido"),
  }

  println!();
  print!("Tecle <Enter> para encerrar... ");
  io::stdout().flush().expect("Falha ao limpar o buffer");
  let mut _entra = String::new();
  io::stdin().read_line(&mut _entra).expect("Falha ao ler a entrada");
}

File: CSHARP/Cap04/c04ex05.cs
// c04ex05.cs

using System;
using static System.Console;

class Program
{
    static void Main(string[] args)
    {
        int Mes;

        Write("Entre com um numero equivalente a um mes: ");
        Mes = int.Parse(ReadLine());

        switch (Mes)
        {
            case 1:
                WriteLine("Janeiro");
                break;
            case 2:
                WriteLine("Fevereiro");
                break;
            case 3:
                WriteLine("Marco");
                break;
            case 4:
                WriteLine("Abril");
                break;
            case 5:
                WriteLine("Maio");
                break;
            case 6:
                WriteLine("Junho");
                break;
            case 7:
                WriteLine("Julho");
                break;
            case 8:
                WriteLine("Agosto");
                break;
            case 9:
                WriteLine("Setembro");
                break;
            case 10:
                WriteLine("Outubro");
                break;
            case 11:
                WriteLine("Novembro");
                break;
            case 12:
                WriteLine("Dezembro");
                break;
            default:
                WriteLine("Mes invalido");
                break;
        }

        Write("\nAperte <Enter> para encerrar... ");
        ReadLine();
    }
}
